### ISSUE TITLE
Hologuide

### ISSUE THREAD

**Issue by CTH999** [LemADEC/WarpDrive#312]
This mod has a wiki that can be hard to understand. It would be nice if there was an in-game guide to help you with it. (Especially the accelerator)

Also, a video guide would be nice, but not needed!

However, this is just a suggestion. It's your mod, so your decision!

**Comment by LemADEC** [comment]
What's hard to understand?
What would make it easier to understand in game?

**Comment by HyperMan19680** [comment]
If it’s a multi block, maybe a hologram-like guide to help assemble it?

**Comment by LemADEC** [comment]
The accelerator is a free-form multi-block, it's up to the player to find the build he/she prefers.
In the long run, there'll be a shipyard allowing to rebuild multi-blocks from schematics, but I'm not sure that answer your question here?

**Comment by CTH999** [comment]
No, I've literally been trying to get the accelerator to work, and I can't get it to work. I don't know if it's me being stupid, the wiki, a bug, or what. I suspect its the first one. But, if it's in the game, it can explain the basics. It's just a suggestion though!

Though, if the shipyard has pre-fabricated partial accelerators. Highly inefficient ones, but still working. You (A.K.A. Me, can fabricate an inefficient one, and get the idea of what to do for it. However, that's just an idea as well!

**Comment by LemADEC** [comment]
Accelerator is only enabled on the official server. It won't do anything in single player, as explained in the mods change logs.
And yes, we would only provide a very inefficient schematic of an accelerator there.

**Comment by CTH999** [comment]
That's good by me! Thanks!

Also, It would be nice If I could use it in creative to experiment with it, So, would It be possible to make it not need/ auto-download/ embed the mod in it?

Just wonding!

**Comment by CTH999** [comment]
Thanks!
